Cargo.toml

```diff
@@ -31,6 +33,7 @@ serde_json = "1.0"
 [dependencies.reqwest]
 version = "0.11"
 optional = true
+default-features = false
 features = ["blocking", "json"]
 
 [dependencies.ureq]
```

Comment:
Please add `default-features = false` and `features = ["gzip", "json"]` for the `ureq` client